data/en/qcas_0018.json

```diff
@@ -0,0 +1,739 @@
+                                "machinery-equipment-acquisitions-answer-1",
+                                "machinery-equipment-acquisitions-answer-2",
+                                "machinery-equipment-acquisitions-answer-3",
+                                "machinery-equipment-acquisitions-answer-4",
+                                "machinery-equipment-acquisitions-answer-5",
+                                "machinery-equipment-acquisitions-answer-6"
+                            ],
+                            "titles": [{
+                                "value": "Total value of acquisitions"
+                            }]
+                        }
+                    },
+                    {
+                        "type": "CalculatedSummary",
+                        "id": "proceeds-playback",
+                        "titles": [{
+                            "value": "We calculate the total value of proceeds from disposals entered to be %(total)s. Is this correct?"
+                        }],
+                        "calculation": {
+                            "calculation_type": "sum",
+                            "answers_to_calculate": [
+                                "machinery-equipment-proceeds-answer-1",
+                                "machinery-equipment-proceeds-answer-2",
+                                "machinery-equipment-proceeds-answer-3",
+                                "machinery-equipment-proceeds-answer-4",
+                                "machinery-equipment-proceeds-answer-5",
+                                "machinery-equipment-proceeds-answer-6"
+                            ],
+                            "titles": [{
+                                "value": "Total value of proceeds from disposals"
+                            }]
+                        }
+                    }, {
+                        "id": "any-significant-changes-block",
+                        "title": "Significant changes",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "any-significant-changes-question",
+                            "title": "Did any significant changes occur to the total investment for {{ metadata['trad_as_or_ru_name'] }}?",
+                            "definitions": [{
+                                "title": "What constitutes a significant change?",
+                                "content": [{
+                                    "description": "What constitutes a ‘significant change’ is dependent on your own interpretation in relation to {{ metadata['trad_as_or_ru_name'] }}’s figures from the previous reporting period and the same reporting period last year."
+                                }]
+                            }],
+                            "type": "General",
+                            "answers": [{
+                                "id": "answer1938",
```

Comment:
is this right?

Reply:
Its not wrong, but ill change to make it meaningful, missed this one.